app/styles/modules/_settings.scss

```diff
@@ -759,8 +759,10 @@ section.modal-panel {
         /*minimum width required for the button to look good without occupying too much space*/
         /*is also the default computed width on desktop screen*/
         min-width: 100px;
+        overflow: hidden;
```

Comment:
@hritvi do you also need to use `text-overflow: ellipsis;`?

Example: https://github.com/mozilla/notes/blob/19632423890f6de87cb5f37ec8fe571a2bc0479b/src/sidebar/static/scss/footer.scss#L43